`ws` node module throws warnings when webpack'd

`webpack` step fails at ToT after a clean (rm node_modules, npm install, ./tools/sigh) today with error below. It's not immediately clear to me if this is due to a storageNG ramdisk dep, or devtools, or something else, see error detail below. Will seek thoughts from @piotrswigon but FYI @shans 

```
> arcs@0.0.0 build:webpack-tools /usr/local/google/home/wkorman/workspace/arcs
> webpack --config ./src/tools/webpack.config.js

Hash: e278cb663a1b8aa70f39
Version: webpack 4.41.2
Time: 11125ms
Built at: 11/22/2019 11:48:30 AM
        Asset     Size  Chunks             Chunk Names
bundle-cli.js  938 KiB       0  [emitted]  bundle-cli
Entrypoint bundle-cli = bundle-cli.js
 [13] external "crypto" 42 bytes {0} [built]
 [15] external "fs" 42 bytes {0} [built]
 [16] external "path" 42 bytes {0} [built]
 [18] external "util" 42 bytes {0} [built]
 [21] external "stream" 42 bytes {0} [built]
 [22] external "events" 42 bytes {0} [built]
 [28] external "os" 42 bytes {0} [built]
 [32] external "buffer" 42 bytes {0} [built]
 [54] external "assert" 42 bytes {0} [built]
 [56] external "http" 42 bytes {0} [built]
 [57] external "url" 42 bytes {0} [built]
 [96] external "https" 42 bytes {0} [built]
 [97] external "net" 42 bytes {0} [built]
[104] (webpack)/buildin/harmony-module.js 573 bytes {0} [built]
[105] ./build/tools/bundle-cli.js + 173 modules 1.65 MiB {0} [built]
      | ./build/tools/bundle-cli.js 1.34 KiB [built]
      | ./build/tools/bundle.js 3.85 KiB [built]
      | ./build/platform/loader.js 379 bytes [built]
      | ./build/runtime/manifest.js 54.5 KiB [built]
      | ./build/platform/assert-node.js 424 bytes [built]
      | ./build/runtime/flags.js 1.92 KiB [built]
      | ./build/runtime/interface-info.js 17.4 KiB [built]
      | ./build/runtime/schema.js 8.05 KiB [built]
      | ./build/runtime/id.js 4.18 KiB [built]
      | ./build/runtime/storageNG/drivers/driver-factory.js 2.04 KiB [built]
      | ./build/runtime/recipe/comparable.js 1.78 KiB [built]
      | ./build/runtime/storageNG/storage-key.js 804 bytes [built]
      | ./build/runtime/storageNG/store.js 2.88 KiB [built]
      | ./build/runtime/type.js 24 KiB [built]
      | ./build/runtime/recipe/type-checker.js 13.3 KiB [built]
      |     + 159 hidden modules
    + 91 hidden modules

WARNING in ./node_modules/ws/lib/buffer-util.js
Module not found: Error: Can't resolve 'bufferutil' in '/usr/local/google/home/wkorman/workspace/arcs/node_modules/ws/lib'
 @ ./node_modules/ws/lib/buffer-util.js
 @ ./node_modules/ws/lib/sender.js
 @ ./node_modules/ws/index.js
 @ ./build/platform/devtools-channel-node.js
 @ ./build/devtools-connector/devtools-connection.js
 @ ./build/devtools-connector/devtools-arc-inspector.js
 @ ./build/runtime/runtime.js
 @ ./build/runtime/storageNG/drivers/ramdisk.js
 @ ./build/runtime/manifest.js
 @ ./build/tools/bundle.js
 @ ./build/tools/bundle-cli.js

WARNING in ./node_modules/ws/lib/validation.js
Module not found: Error: Can't resolve 'utf-8-validate' in '/usr/local/google/home/wkorman/workspace/arcs/node_modules/ws/lib'
 @ ./node_modules/ws/lib/validation.js
 @ ./node_modules/ws/lib/sender.js
 @ ./node_modules/ws/index.js
 @ ./build/platform/devtools-channel-node.js
 @ ./build/devtools-connector/devtools-connection.js
 @ ./build/devtools-connector/devtools-arc-inspector.js
 @ ./build/runtime/runtime.js
 @ ./build/runtime/storageNG/drivers/ramdisk.js
 @ ./build/runtime/manifest.js
 @ ./build/tools/bundle.js
 @ ./build/tools/bundle-cli.js

```

--- a/src/tools/webpack.config.js
+++ b/src/tools/webpack.config.js
@@ -26,6 +26,7 @@ module.exports = {
     filename: '[name].js',
     path: path.resolve(__dirname, `../../dist/tools`)
   },
+  externals: ['bufferutil', 'utf-8-validate'],
   plugins: [
     new webpack.NormalModuleReplacementPlugin(
       // Replace all the web variants with node ones.

